### ISSUE TITLE
timer problem

### ISSUE THREAD

**Issue by zsole2** [kizniche/mycodo_python#16]
I cannot set the timers. 

Apparently there is not much to set: 
add relay, name timer, select relay, set timer settings, create....
Then I get: 
"Internal Server Error
The server encountered an internal error and was unable to complete your request. Either the server is overloaded or there is an error in the application."

I started the daemon in debug mode as suggested, but there is no /tmp/mycodo.log file.

**Comment by kizniche** [comment]
Thanks for the feedback.

That should be /var/log/mycodo.log

I found the error and fixed the timer issue in commit ae1e54c

**Comment by kizniche** [comment]
Also, as a correction, for this type of issue you would want to start flask in debug mode instead of the daemon to see why the web server is producing the error.

```
sudo /etc/init.d/apache2 stop
sudo ~/mycodo_python/mycodo/mycodo_flask.py -d
```
